Scripts/DCS-BIOS/lib/modules/aircraft_modules/F-4E.lua

```diff
@@ -1452,6 +1452,37 @@ local AIRCRAFT_EFFECTS_DEVICE_ID = 63
+F_4E:definePushButton("WSO_TGT_DESIGNATOR_STOW", TARGET_DESIGNATOR_DEVICE_ID, 3005, 2754, TARGET_DESIGNATOR_SET_CONTROL, "Stow/Unstow TGP")
+F_4E:defineToggleSwitch("WSO_TGT_DESIGNATOR_LASER_READY", TARGET_DESIGNATOR_DEVICE_ID, 3006, 2755, TARGET_DESIGNATOR_SET_CONTROL, "Arm/Disarm Laser")
+F_4E:definePushButton("WSO_TGT_DESIGNATOR_POWER_ON", TARGET_DESIGNATOR_DEVICE_ID, 3007, 2756, TARGET_DESIGNATOR_SET_CONTROL, "Power On/Off TGP")
+F_4E:definePushButton("WSO_TGT_DESIGNATOR_BIT", TARGET_DESIGNATOR_DEVICE_ID, 3008, 2737, TARGET_DESIGNATOR_SET_CONTROL, "Next BIT")
+F_4E:definePotentiometer("WSO_TGT_DESIGNATOR_LIGHT_BRIGHTNESS", TARGET_DESIGNATOR_DEVICE_ID, 3009, 2738, { 0, 1 }, TARGET_DESIGNATOR_SET_CONTROL, "Panel Light Brightness")
+F_4E:definePushButton("WSO_TGT_DESIGNATOR_REJECT_OVERRIDE", TARGET_DESIGNATOR_DEVICE_ID, 3010, 2739, TARGET_DESIGNATOR_SET_CONTROL, "Reject/Override Range")
+F_4E:defineMultipositionSwitch("WSO_TGT_DESIGNATOR_ACQUISITION", TARGET_DESIGNATOR_DEVICE_ID, 3011, 2740, 3, 0.5, TARGET_DESIGNATOR_SET_CONTROL, "Acquisition Mode")
+F_4E:defineToggleSwitch("WSO_TGT_DESIGNATOR_WRCS_OUT", TARGET_DESIGNATOR_DEVICE_ID, 3012, 2757, TARGET_DESIGNATOR_SET_CONTROL, "Enable/Disable WRCS integration")
+F_4E:defineToggleSwitch("WSO_TGT_DESIGNATOR_GO_MALF", TARGET_DESIGNATOR_DEVICE_ID, 3013, 2758, TARGET_DESIGNATOR_SET_CONTROL, "(no function)")
+F_4E:defineToggleSwitch("WSO_TGT_DESIGNATOR_INS_OUT", TARGET_DESIGNATOR_DEVICE_ID, 3014, 2759, TARGET_DESIGNATOR_SET_CONTROL, "Enable/Disable INS integration")
+
+-- Laser Coder Control
+F_4E:definePushButton("WSO_LASER_CODE_ENTER", TARGET_DESIGNATOR_DEVICE_ID, 3015, 2504, TARGET_DESIGNATOR_SET_CONTROL, "Enter Laser Code")
+F_4E:definePushButton("WSO_LASER_CODE_SET_THOUSANDS", TARGET_DESIGNATOR_DEVICE_ID, 3019, 2500, TARGET_DESIGNATOR_SET_CONTROL, "Next Laser Code (thousands)")
+F_4E:definePushButton("WSO_LASER_CODE_SET_HUNDREDS", TARGET_DESIGNATOR_DEVICE_ID, 3018, 2501, TARGET_DESIGNATOR_SET_CONTROL, "Next Laser Code (hundreds)")
+F_4E:definePushButton("WSO_LASER_CODE_SET_TENS", TARGET_DESIGNATOR_DEVICE_ID, 3017, 2502, TARGET_DESIGNATOR_SET_CONTROL, "Next Laser Code (tens)")
+F_4E:definePushButton("WSO_LASER_CODE_SET_ONES", TARGET_DESIGNATOR_DEVICE_ID, 3016, 2503, TARGET_DESIGNATOR_SET_CONTROL, "Next Laser Code (ones)")
+
+-- Range Indicator
+F_4E:definePushButton("PILOT_RANGE_INDICATOR_TEST", TARGET_DESIGNATOR_DEVICE_ID, 3020, 2764, TARGET_DESIGNATOR_SET_CONTROL, "Test Display")
+F_4E:definePotentiometer("PLT_RANGE_INDICATOR_BRIGHTNESS", TARGET_DESIGNATOR_DEVICE_ID, 3021, 1211, { 0, 1 }, TARGET_DESIGNATOR_SET_CONTROL, "Display Brightness")
+F_4E:definePotentiometer("PLT_RANGE_INDICATOR_DESIRED_RANGE", TARGET_DESIGNATOR_DEVICE_ID, 3023, 1212, { 0, 1 }, TARGET_DESIGNATOR_SET_CONTROL, "Desired Release Range")
+F_4E:defineMultipositionSwitch("PILOT_RANGE_INDICATOR_MODE", TARGET_DESIGNATOR_DEVICE_ID, 3022, 1210, 3, 0.5, TARGET_DESIGNATOR_SET_CONTROL, "Range Mode")
```

Comment:
```suggestion
F_4E:define3PosTumb0To1("PLT_RANGE_INDICATOR_MODE", TARGET_DESIGNATOR_DEVICE_ID, 3022, 1210, 3, TARGET_DESIGNATOR_SET_CONTROL, "Range Mode")
```

Reply:
Just realized I left the 3 in the function signature, but it can also be removed